### DISCUSSION TITLE
How can we delete keda HPA after HttpScaledObject deleted

### DISCUSSION THREAD

**Discussion by surachai-tep** [kedacore/http-add-on#428] [Q&A] [answered]
Hi,
 I successfully deploy httpscaledobject and be able to scale to zero pod. But when I don't want it, I deleted httpscaledobject, my pod still be zero even I update replicas in deployment to 1.

 I also found there's one keda-hpa-xxx that created along with this deployment. I tried to delete this hpa but seems it back to be created again.
 Question: How can I clean up the httpscaledobject and related components?

Thank you

**Comment by arschles** [comment, marked as answer]
@surachai-tep the KEDA HTTP Addon creates a `ScaledObject` behind the scenes, and KEDA itself then creates the HPA from that. Is there still a `ScaledObject` after you deleted the `HTTPScaledObject`?

**Reply by surachai-tep** [reply]
Thank you very much @arschles,  delete ScaledObject help!
